client/js/components/statement/assessmentTable/ConsolidateModal.vue

```diff
@@ -481,7 +481,7 @@ export default {
             location.reload()
           }
         })
-        .catch(e => handleResponseMessages(e.response.data.meta))
+        .catch(e => handleResponseMessages(Translator.trans(e.response.data.meta)))
```

Comment:
This can't work because `handleResponseMessages` expects an object as argument, not a string. Did you test this?

Reply:
Could it be, that the Messages comming from the response are already translated and don't need any special handling. I am pretty sure, but please doublecheck

Reply:
I think if `handleResponseMessages` expects an object as argument then we can parse object then stringify:
```
.catch((e) => {
   const metaObject = JSON.parse(e.response.data.meta)
   const translateMetaString = Translator.trans(JSON.stringify(metaObject))
   const responseMeta = JSON.parse(translateMetaString)
   handleResponseMessages(responseMeta)
})
```
Is that right?

Reply:
no. why do You want to transform it? Please have a look in demosplan-ui, how the params are handled there.

Reply:
Or we can make sure the both Concerns with condition:
```
.catch(e => {
  const responseMeta = Translator.trans(e.response.data.meta)
  if (typeof responseMeta === 'string') {
    handleResponseMessages(JSON.parse(responseMeta))
  } else {
    handleResponseMessages(responseMeta)
  }
})
```

Reply:
https://github.com/demos-europe/demosplan-core/pull/3467/commits/fcab2595cb831845c83b6692d30fe2e2828b4c39